### ISSUE TITLE
Bot mute

### ISSUE THREAD

**Issue by OGContent** [Minehut/Meta#635]
### Bug report

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] I am NOT reporting a security vulnerability
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ x] This issue does not contain any sensitive information

**Platform**

discord
**Describe the bug**

when you get muted and leave the discord and join it again the auto unmute wont work on you
**To reproduce**
Steps to reproduce the behaviour:
1. Get muted
2. Leave the discord server
3. Join the discord server again
4. wait 6 hours or the time that you got muted for and you wont get auto unmuted

**Expected behaviour**

to get auto unmuted
**Device Information**

- Minecraft client (and version): 
- Web browser (and version): 
- Operating system (and version): 
- Minehut server name: 

**Screenshots**

**Additional context**

---
**Try to reproduce this bug and let us know in the comments if you can reproduce it or not. Make sure to include your relevant device information!**

**Comment by LiftenMH** [comment]
Please make these things more adequate. Also this isn't a security thing whilst you didn't check it as not one.

**Comment by emopedia** [comment]
Yeah this has happened to me multiple times. especially with me getting banned and unbanned while muted, once my mute "expires" i'll have to appeal to get it removed
